src/prefect/client/schemas/responses.py

```diff
@@ -420,6 +429,22 @@ class DeploymentResponse(ObjectBaseModel):
         description="Current status of the deployment.",
     )
 
+    def as_related_resource(self, role: str) -> "RelatedResource":
```

Comment:
Is it worthwhile to include a default value here?
```suggestion
    def as_related_resource(self, role: str = "deployment") -> "RelatedResource":
```